### ISSUE TITLE
A11y_Azure App Service extension for VS Code_Azure App services_icon_Keyboard: The tool tip is provided when we hover over "Azure App Services" in the list using mouse, but is not accessible using keyboard arrow keys.

### ISSUE THREAD

**Issue by v-gitsay** [microsoft/vscode-azureappservice#2135]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#GH_AzureAppServiceextensionforVSCode_Win32_Sep2021;#Azure Tools for VS Code;#DesktopApp;#WCAG2.1.1;#AINotInScope;#FTP;#Keyboard;

### Environment Details:
Application Name: Azure App Service extension for VS Code
Screen Reader: NVDA
Operating system: Windows 10

### Repro Steps:

1. Launch VS Code application.
2. Tab till Extensions in the Navigation bar and press enter.
3. Tab to search box.
4. Type "Azure App Services" in search bar and Tab to the "Azure App Services" in the resulting list.
5. Check if the tool tip is accessible for "Azure App Services" in the list using keyboard .

### Actual:
The tool tip is provided when we hover over "Azure App Services" in the list using mouse, but is not accessible using keyboard arrow keys.

### Expected:
Tool tip should be accessible for "Azure App Services" in the list using keyboard arrow keys.

### User Impact:
Keyboard users will not get complete information about the list item "Azure App Services" .

**Comment by nturinski** [comment]
An extension cannot control the extension view and tooltips.  This issue should be filed on VS Code: https://github.com/microsoft/vscode/issues

**Comment by v-gitsay** [comment]
External bug: https://github.com/microsoft/vscode/issues/132344

**Comment by MicroFish91** [comment]
**Informational summary**:
  1) User has found a potential bug issue in the Extension view
  2) One of our team members identified it as an external / upstream issue
  3) Original user has escalated the issue to VS Code team and mentioned it here for tracking
  4) Tracked issue states that VS Code team has had a similar issue in the past where they have basically stated that this issue is such a rare exception that there is very low demand for a feature add, see here for a concise breakdown:  https://github.com/Microsoft/vscode/issues/52016#issuecomment-641096443

**Comment by v-gitsay** [comment]
Lowering the Severity to Sev3 as per discussion with SME " this issue is not for screen reader users but for keyboard users as it is not accessible by keyboard focus"

**Comment by jinglouMSFT** [comment]
@v-gitsay since the parent external bug has been moved to Sev 3 and you have also lowered the severity of this bug to Sev3, could you please also remove the accessibility-sla tag? This bug shouldn't show up in my S360 tracking but it is.  Thanks.

**Comment by v-gitsay** [comment]
we have changed the severity of the bug to A11ysev3 it will reflect in S360 in some days

**Comment by keerthiduvvuri** [comment]
GithubTags:#ResolvedByTT

**Comment by Mana-swi** [comment]
#a11ydb-hover-only-content;
